src/fw/process_management/app_manager.c

```diff
@@ -214,7 +214,7 @@ static size_t prv_get_app_segment_size(const PebbleProcessMd *app_md) {
 #if CAPABILITY_HAS_JAVASCRIPT
       if (app_md->is_rocky_app) {
         // on Spalding, we didn't have enough applib padding to guarantee both,
-        // 4.x native app heap + JerryScript statis + increased stack for Rocky.
+        // 4.x native app heap + JerryScript status + increased stack for Rocky.
```

Comment:
I thought about this initially, but I've reconsidered

```suggestion
        // 4.x native app heap + JerryScript static + increased stack for Rocky.
```